### DISCUSSION TITLE
How can I get the size of the backups and snapshots ?

### DISCUSSION THREAD

**Discussion by camillebernier** [vmware-tanzu/velero#5466] [Community support Q&A]
Hello,
I'm using Velero with Restic (for the volume snapshots) and Minio. 
All installed with Helm on Kubernetes.

Is there any way to get the size of the Backups and Snapshots (done with Restic)?
And to have details on each backup for example: 
- Size when the backup is incremental
- Size for a full backup
- Size for a snapshot
- Size before and after compression 
I can't find a way to do this, except by making a backup of a single namespace and its resources, and check the size of my bucket after.

Thanks

**Comment by RogerSik** [comment]
Also looking for this. This would make troubleshooting easier to understand why the s3 bucket is growing so fast.

**Comment by RogerSik** [comment]
@Camiyo for a now solution. It is possible to use the restic client in the kubernetes pod (to avoid the setup and configuration) with the argument "restic stats" to get a size of backups. This needs to be called separately for every namespace (above example command for the namespace deconz)

```
kubectl exec -it deployment/velero -- restic -r "s3:https://minio.example.org/velero/restic/deconz" stats
```

The requested password is: *static-passw0rd*
